streams/src/test/java/org/apache/kafka/streams/integration/KStreamKTableJoinIntegrationTest.java

```diff
@@ -62,7 +67,7 @@
 @Category({IntegrationTest.class})
 public class KStreamKTableJoinIntegrationTest {
     private static final int NUM_BROKERS = 1;
-    private static final long COMMIT_INTERVAL_MS = 300L;
+    private static final long COMMIT_INTERVAL_MS = 1000L;
```

Comment:
Since we already changed the checking when cache is turned on, I'm wondering do we still want to increase the commit interval here?

Reply:
It's just to make it less likely that we hit a flush in between. Can also revert (btw: in a local test, I set it to `1` and the test still passed -- but this would imply that we flush very often and I guess we want to test the case without flushing in-between if posssible).